docs/scos/dev/guidelines/keeping-a-project-upgradable/code-upgrader-supported-extension-scenarios.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Spryker Code Upgrader supported extensions scenarios
+description: Spryker extensions scenarios covered by Spryker Code Upgrader
+last_updated: Mar 13, 2023
+template: concept-topic-template
+related:
+  - title: Plugins registration
+    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+  - title: Event subscribers registration
+    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+  - title: Modules configuration
+    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.html
+---
+
+## Introduction
+
+Spryker Code Upgrader implements code changes on the project level
+
+Code changes on the project levels can be delivered to customer projects through the manifest files.
+The manifests cover a list of the ways how the code changes can be done.
+This article aims to describe the ways of plugin registration and module configuration that can be covered by manifests.
+
+To ensure the successful delivery of Spryker updates, we recommend using the extension points that exist in the article.
+
+{% info_block infoBox "Exceptions" %}
```

Comment:
I wouldn't call it "exceptions". It is how a customer can extend the Spryker project without blocking upgradability.

Reply:
Renamed to "Warnings"
The info box contains info "what happens if customer skipped the rules" - it is kind of "not usual case"